Makefile

```diff
@@ -36,7 +36,7 @@ install_requirements:
 	pip install -r requirements.txt
```

Comment:
This isn't quite related to this PR but install requirements seems to fail for me
![image](https://user-images.githubusercontent.com/37023871/125147563-2dd03e00-e0fa-11eb-84d3-0c42c037e8e1.png)

Could you fix this as well?